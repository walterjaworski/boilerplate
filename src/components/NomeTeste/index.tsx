import * as S from './styles'

const NomeTeste = () => (
  <S.Wrapper>
    <h1>NomeTeste</h1>
  </S.Wrapper>
)

export default NomeTeste
