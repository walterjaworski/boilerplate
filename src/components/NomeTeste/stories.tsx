import { Story, Meta } from '@storybook/react/types-6-0'
import NomeTeste from '.'

export default {
  title: 'NomeTeste',
  component: NomeTeste
} as Meta

export const Default: Story = () => <NomeTeste />
