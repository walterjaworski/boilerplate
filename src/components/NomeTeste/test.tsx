import { render, screen } from '@testing-library/react'

import NomeTeste from '.'

describe('<NomeTeste />', () => {
  it('should render the heading', () => {
    const { container } = render(<NomeTeste />)

    expect(
      screen.getByRole('heading', { name: /NomeTeste/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
