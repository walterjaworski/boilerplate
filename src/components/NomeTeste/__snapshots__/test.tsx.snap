// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NomeTeste /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    NomeTeste
  </h1>
</main>
`
